.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,16 @@
+rebaseMergeAllowed: true
+squashMergeAllowed: true
+mergeCommitAllowed: false
+branchProtectionRules:
+- pattern: master
+  isAdminEnforced: true
+  requiredApprovingReviewCount: 1
+  requiresCodeOwnerReviews: true
+  requiresStrictStatusChecks: true
+permissionRules:
+  - team: actools-python
+    permission: admin
+  - team: actools
+    permission: admin
+  - team: yoshi-python
+    permission: push
```

Comment:
@JustinBeckwith Do these permission rules get combined with the default python rule or should we explicitly call out all the groups that need access?

Reply:
They are combined.  This is always additive.